### ISSUE TITLE
HC QUERY: User terminology

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#187]
@kevincrepin @ilmartyrk @anettlinno For the HC texts, I'd like to make sure I'm using the right terminology for different types of user: 

For example, is it this? 

- Group/topic *admin*
- Group/topic *initiator* (though presumably we won't often have to refer to them as anything other than "admin", as they won't have any different rights or see anything differently as a result of having been the one that created it?) 
- Group *member*
- Topic *participant* (or would it be "follower"?) - see question below. 

One thing that has confused me a bit when working on the HC texts is the use of the word "follow" in places. As in some places we join a topic, and in some we follow it, (and then in the rights we are "participant"). Am I right in understanding that "Join" is for private topics, whereas "Follow" is for public groups, but in both cases the rights are "Participant"? (Mostly am just asking to be clear for the HC texts, but also wondering if that will create some confusion - personally I was wondering if following is different to joining/participating - i.e. if you have different rights or see different things as a follower versus a full participant).

**Comment by BeccaMelhuish** [comment]
P.S. I see that we have a "Topic owner" label in the designs, and am wondering what this means in practice. I assume it's the person who first started the topic, but I'm wondering if they have any different rights or role to any other admin (and whether more than one person can "own" it, or only the person who made it). And if no different to admin rights/role, should we just stick to the 'admin' label for everyone who has admin level rights (not not confuse things). 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/412baff1-73fb-4a2a-be5a-b70704b9f852)

**Comment by anettlinno** [comment]
Regarding the group/topic admin and owner. We don´t have separate rights to admin or owner, so I think that we should keep it as an **admin** everywhere. 

Regarding the initiator -- I don´t think we use **initiator** anywhere on the platform? We only use it when we talk about person who initiated the topic and in HC I think it´s correct to use it. On the platform we should only be talking about topic/group **admin** as of topic initiator. 

Regarding group members and topic participants. In group we only have two roles -- admin and member. I think it is correct to use **members** in group and **participants** in topic. BUT in order to keep it clear we should keep the consistency and change the **participant** role in the group also to a **member**. In topic, let´s keep it still **participant**.

Regarding **Join group** and **Follow topic**. Again, we should keep consistency here as well and choose either one or the other. I would prefer to use **Join** as if I have joined the group, I´m a member of the group and if I have **joined the topic** (where you could mentally add **discussion**), it makes me feel that I´m a participant in the discussion of this topic.

**Comment by anettlinno** [comment]
Triage 90. In translation files we have "Creator" but we don´t use it anywhere on the platform. This should be mutually agreed, what terminology we use -- either "initiator" or "creator". 

We have discussed that we should create third layer in group roles -- similar to editor role in topics. 
So that:

- Admin has highest rights who can do everything (add topics, invite members, delete group, change group name and image, intro text, creating rules, group moderation etc);
- Editor can add topics in the group and in private topics could also invite members;
- Member can participate in topics.

Admin user should have rights to remove topics and members from the group. If there is a use case where Editor adds topic which is in contradiction with good will group rules, s/he could just remove the topic or member without having admin rights in topic.

**Comment by anettlinno** [comment]
In redesign we have added roles in topic view: 
![image](https://github.com/citizenos/citizenos-fe/assets/45175679/3f5333d0-bdcf-47ba-8202-933a9e62a423)

We could change those admin, editor or participant. 

Under the drop-down menu in redesign there is "Unfollow topic" and this could be changed to "Leave topic" as in current version
![image](https://github.com/citizenos/citizenos-fe/assets/45175679/162701b8-2d1f-4bd6-b158-96ef41a02593)

**Comment by kevincrepin** [comment]
One point about following/unfollow: the idea was that a user could follow a topic and it would therefor show up in their feed, without having to "participate" (discuss, ideate, vote). 

For public topics where a personal invite might not happen I think this is an interesting extra "role" which serves less active users to still keep following topics of their interest.

**Comment by BeccaMelhuish** [comment]
Notes from meeting with Anett: 
- Let's not keep follower, but only join.  
- On the platform, let's not keep topic owner or creator, but only admin. On HC can call them topic initiators. 
- Let's add the editor role to group roles, as per Anett's explanation above. 
- The lowest rights for groups will be called members, but for topics will be participants. 

@kevincrepin could you update the Figma in all places these words/roles appear?

**Comment by kevincrepin** [comment]
So users are joining topics and joining groups? We'd make no difference in wording for these different actions?

**Comment by BeccaMelhuish** [comment]
@kevincrepin yes I think so, unless there's another word for topics that works better, but I can't think of one myself.. Do you see a problem in having them both the same? It seems to make sense to me. 

And just to clarify what Anett and I had understood from your suggestions re 'Followers' - that there would be two options - you could either follow or join a topic, depending on how active you wanted to be. But I'm thinking now you had perhaps meant that you 'Follow' a topic, but 'Join' a group? 

I know Anett was keen on Join for topics though (in her [comment above](https://github.com/citizenos/citizenos-fe/issues/187#issuecomment-1733323816)), as a more active word and to keep the consistency in wording with groups. And I feel the same myself. 

Would you advise against?

**Comment by kevincrepin** [comment]
Yep Becca, how you explained it there is how we meant it. I've been going back and forth with it and it can be join as well. It's not a huge thing, I think.

**Comment by anettlinno** [comment]
I´m trying to sum it up about what needs to be checked in the texts @BeccaMelhuish :) Please correct me if I missed anything

- [x] Join topic and join group (instead of follow) 
- [x] Topic participant (instead of follower)
- [ ] Group member instead of participant
- [ ] Add editor role in the group with following permission (currently still admin and participant):

- Admin has highest rights who can do everything (add topics, invite members, delete group, change group name and image, intro text, creating rules, group moderation etc);
- Editor can add topics in the group and in private topics could also invite members;
- Member can participate in topics.

![image](https://github.com/citizenos/citizenos-fe/assets/45175679/22eb2dce-2d6e-4dce-8280-5e651cd1526f)

- [ ] Replace "Owner" in the texts to "Admin"

![image](https://github.com/citizenos/citizenos-fe/assets/45175679/58c68c96-329c-46ed-ab76-04ad3bb7025f)

**Comment by anettlinno** [comment]
> And just to clarify what Anett and I had understood from your suggestions re 'Followers' - that there would be two options - you could either follow or join a topic, depending on how active you wanted to be. But I'm thinking now you had perhaps meant that you 'Follow' a topic, but 'Join' a group?
> 

@kevincrepin regarding following and joining the topic. IMHO we don´t need to separate Users by their activity. Because if they join the topic and they only wish to receive notifications without any further activity, they can still be inactive and receive notifications being joined to topic :)

**Comment by BeccaMelhuish** [comment]
@anettlinno yes these all look right to me! 

Some of this checking I can do myself: if it still says follow/follower anywhere I should be able to find and replace quite easily. Switching group participant to member might be more tricky as will depend on the context (e.g. in drop-down boxes where it's just one word, not so clear in the strings), but I can definitely give it a go and ask @ilmartyrk for help if needed :)

**Comment by BeccaMelhuish** [comment]
@anettlinno update on this: 

- There was no follow/follower in the translation strings so I've ticked those two as done. 
- I went through and switched participant to member when it was related to groups, except the one below. So if @ilmartyrk can advise on/edit that one it can be ticked off too :) 

Just this one I wasn't sure if it related to groups or topics (and therefore should be members or participants): 
![image](https://github.com/citizenos/citizenos-fe/assets/61320526/d6b52409-8a5c-4f88-91d5-a7da0d493e48)

For this 'owner' one, I searched for owner in [here](https://github.com/citizenos/citizenos-fe/edit/redesign/src/assets/i18n/en.json), and it came back with no results. Which makes me think some might be missing from the translation strings @ilmartyrk? It happened [here](https://github.com/citizenos/citizenos-fe/issues/309) too (but I saw this typo was fixed now so closed it)  

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/88837a79-956b-4a2d-b817-0b8baadd1be4)

**Comment by ilmartyrk** [comment]
@anettlinno @BeccaMelhuish I am confused by this issue, there are images from prototype about views that we haven't added yet as we don't have the functionalities so I guess it is not Phase I. Also it's mixed up, with adding extra permissions to groups. I think it should be a Phase II thing rather than trying to cram it into Phase I?

**Comment by anettlinno** [comment]
@BeccaMelhuish if you can be more clear about where you need @ilmartyrk ´s help, would be nice. If terminology matches with what we agreed for Phase I then I also suggest to close this issue now and open new issue when mismatches in platform and design appear.

**Comment by ilmartyrk** [comment]
@BeccaMelhuish check if still relevant
